#!/bin/sh /usr/share/dpatch/dpatch-run
## 206-magic-update-bash.dpatch
##
## DP: Add another shebang path for bash.

@DPATCH@

diff -Naurp file.orig/magic/Magdir/commands file/magic/Magdir/commands
--- file.orig/magic/Magdir/commands	2008-07-15 13:31:02.000000000 +0000
+++ file/magic/Magdir/commands	2008-07-15 13:30:55.000000000 +0000
@@ -58,6 +58,10 @@ o	string/b	#!\ /usr/bin/tcsh	Tenex C she
 # bash shell magic, from Peter Tobias (tobias@server.et-inf.fho-emden.de)
 0	string/b	#!\ /bin/bash	Bourne-Again shell script text executable
 !:mime	text/x-shellscript
+0	string/b	#!\ /usr/bin/bash	Bourne-Again shell script text executable
+!:mime text/x-shellscript
+0	string/b	#!\ /usr/local/bash	Bourne-Again shell script text executable
+!:mime text/x-shellscript
 0	string/b	#!\ /usr/local/bin/bash	Bourne-Again shell script text executable
 !:mime	text/x-shellscript
 
